Add more documentation (#32)
QwantResearch/addresses-importer: importers/bano/README.md

> This importer import the addresses from [BANO]. It requires as first argument, the csv file (you can

Reviewer: I don't like pleaunasms :D

```suggestion
This is the importer for [BANO]. It requires as first argument, the csv file (you can
```

download it from [here](http://bano.openstreetmap.fr/data/)) in which it'll find the data from
[BANO].

## How it works

The script reads the provided CSV file. Each line is then added if it has the following elements:

 * longitude
 * latitude
 * street name
 * house number

## Running it

> You can run it like this:

Reviewer: Shouldn't we still add a little comment about were to find the output?

Author: Good idea!

```bash
$ cargo run --release -- [BANO csv file]
```

> ## Using it as a library

Reviewer: `import_addresses` actually has the following skeleton:

```rust
fn import_addresses<P: AsRef<Path>, T: CompatibleDB>(file_path: P, db: &mut T);
```

The skeleton is the wrong one and `file_path` is called `file` in arguments documentation.
